test/controllers/skill_controller_test.exs

```diff
@@ -15,11 +15,13 @@ defmodule CodeCorps.SkillControllerTest do
   defp put_attributes(payload, attributes), do: payload |> put_in(["data", "attributes"], attributes)
 
   describe "index" do
-    test "lists all entries on index", %{conn: conn} do
+    test "lists first five entries on index", %{conn: conn} do
+      insert_list(6, :skill)
       path = conn |> skill_path(:index)
       json = conn |> get(path) |> json_response(200)
 
-      assert json["data"] == []
+      returned_skills_length = json["data"] |> length
+      assert returned_skills_length == 5
```

Comment:
Yeah, I was thinking about that but couldnt figure out how to discern between the two with only backend. This might be better tackled on the Ember side to just display 5 after it requests all matching skills?